toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1273,23 +1234,21 @@ func createUserWithPassword(installChroot *safechroot.Chroot, user configuration
 			return
 		}
 
-		err = installChroot.UnsafeRun(func() error {
-			return chage(user.PasswordExpiresDays, user.Name)
-		})
+		err = Chage(user.PasswordExpiresDays, user.Name)
```

Comment:
Will this still correctly find the correct /etc/shadow for offline builds? I'd think it would get the setupChroot's copy without the nested chroot call? Do we need to pass `installChrootShadowFile` instead of getting it via `shadow, err = file.ReadLines(ShadowFile)` in `Chage()`?

Reply:
Good catch.

Reply:
Fixed.